WordPress/Classes/ViewRelated/People/PeopleViewController.swift

```diff
@@ -338,27 +339,29 @@ open class PeopleViewController: UITableViewController, NSFetchedResultsControll
 
     // MARK: - No Results Helpers
 
-    fileprivate func refreshNoResultsView() {
+    private func refreshNoResultsView() {
+
+        noResultsViewController.removeFromView()
+
         guard resultsController.fetchedObjects?.count == 0 else {
-            noResultsView.removeFromSuperview()
             return
         }
 
-        noResultsView.titleText = noResultsTitle()
+        noResultsViewController.configure(title: noResultsTitle(),
+                                          image: "wp-illustration-empty-results")
 
-        if noResultsView.superview == nil {
-            tableView.addSubview(withFadeAnimation: noResultsView)
-        }
+        addChildViewController(noResultsViewController)
```

Comment:
When I saw the `didMove(toParent parent: UIViewController?)`, I was curious about the apparent lack of a call to `willMove(toParent parent: UIViewController?)`. 

TIL, however, that it's automatically called here! [See also](https://developer.apple.com/documentation/uikit/uiviewcontroller/1621381-willmove)